fhir-install/src/main/docker/ibm-fhir-schematool/run.sh

```diff
@@ -181,15 +181,15 @@ function _call_db2 {
 
     # since we are generating, we can debug this... with set +x
     set -x
-    /opt/java/openjdk/bin/java -jar ${SCHEMA_TOOL_LOCATION}/fhir-persistence-schema-*-cli.jar \
+    /opt/java/openjdk/bin/java -Dlog.dir=${SCHEMA_TOOL_LOCATION}/workarea -jar ${SCHEMA_TOOL_LOCATION}/fhir-persistence-schema-*-cli.jar \
         --prop "db.host=${DB_HOSTNAME}" \
         --prop "db.port=${DB_PORT}" \
         --prop "db.database=${DB_NAME}" \
         --prop "user=${DB_USER}" \
         --prop "password=${DB_PASSWORD}" \
         ${SSL_STANZA} \
         --db-type db2 \
-        ${INPUT} 2>&1 | tee out.log
+        ${INPUT} 2>&1 | tee ${SCHEMA_TOOL_LOCATION}/workarea/out.log
     echo "$?" > response_code
     set +x
 }
```

Comment:
Update lines 218 and 226 as well (for postgres).

Reply:
Updated per comments.